pkg/eigenState/operatorDirectedRewardSubmissions/operatorDirectedRewardSubmissions.go

```diff
@@ -80,6 +76,14 @@ func (odrs *OperatorDirectedRewardSubmissionsModel) GetModelName() string {
 	return "OperatorDirectedRewardSubmissionsModel"
 }
 
+func (odrs *OperatorDirectedRewardSubmissionsModel) NewSlotID(blockNumber uint64, transactionHash string, logIndex uint64, rewardHash string, strategyIndex uint64, operatorIndex uint64) types.SlotID {
+	if odrs.globalConfig.ChainIsOneOf(config.Chain_Holesky, config.Chain_Preprod) && blockNumber < 3118055 {
```

Comment:
We should be pulling the block number from the config rather than hardcoding IMO.